src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/issuegenericletter/IssueGenericLetterAboutToStartHandler.java

```diff
@@ -93,6 +93,10 @@ private List<DynamicListItem> getDocumentsListOptions(List<? extends AbstractDoc
             for (var document : documents) {
                 String documentFileName = document.getValue().getDocumentFileName();
                 listOptions.add(new DynamicListItem(documentFileName, documentFileName));
+                if (document.getValue().getEditedDocumentLink() != null) {
+                    String editedDocumentFileName = document.getValue().getEditedDocumentLink().getDocumentFilename();
+                    listOptions.add(new DynamicListItem(editedDocumentFileName, editedDocumentFileName));
+                }
```

Comment:
we need to add a unit/integration test to cover this ?

Reply:
Have added the test in new commit.